sros2/setup.py

```diff
@@ -66,7 +66,9 @@ def package_files(directory):
             ':CreatePermissionVerb',
             'distribute_key = sros2.verb.distribute_key:DistributeKeyVerb',
             'generate_artifacts = sros2.verb.generate_artifacts:GenerateArtifactsVerb',
-            'generate_policy = sros2.verb.generate_policy:GeneratePolicyVerb',
+            # TODO(ivanpauno): Reactivate this after having a way to instronspect
+            # security context names in rclpy
```

Comment:
Do we have an issue in rclpy that we can use for tracking this?

Reply:
+1, TODOs in code usually don't get revisited if there isn't an associated ticket (same for other TODOs below)

Spelling: instronspect -> introspect

Reply:
See https://github.com/ros2/sros2/issues/182, https://github.com/ros2/rclpy/issues/528, https://github.com/ros2/rclpy/issues/529, https://github.com/ros2/sros2/issues/183.

Reply:
Can we add a reference to https://github.com/ros2/rclpy/issues/529 in this comment, please?